Sources/Kumo/Blobs/Storage/FileSystem.swift

```diff
@@ -48,7 +48,16 @@ class FileSystem: StorageLocation {
     func acquire<D: _DataRepresentable>(fromPath path: URL, origin url: URL, arguments: D._RepresentationArguments) throws -> D? {
         guard let data = backingManager.contents(atPath: path.path) else { return nil }
         let newPath = parentDirectory.appendingPathComponent(murmur3_32(url.absoluteString))
-        try backingManager.moveItem(at: path, to: newPath)
+
+        do {
+            try backingManager.moveItem(at: path, to: newPath)
+        } catch {
+            let nsError = error as NSError
+            if nsError.code != 516 || nsError.domain != NSCocoaErrorDomain {
```

Comment:
I would make these static extensions of `NSError`—a `var` for the error code named by what it represents, and a function that does this check here for determining whether it's a salient error

Reply:
Good idea, done